vector-im/element-android
# Create the DM when sending an event

## Type of change


## Content

Depends on #6051 

Create the DM room when an event is sent from the local room timeline (text, image, audio, poll, location sharing...)

## Motivation and context

Continue #5525 

## Screenshots / GIFs

||Sender|Receiver|
|-|-|-|
|By text|https://user-images.githubusercontent.com/11990514/178769378-f9348056-c725-418a-a830-f0afbad358bd.mp4|https://user-images.githubusercontent.com/11990514/178769798-f7f90ce8-d410-47c2-9a07-672bd4278e06.mp4|
|By image|https://user-images.githubusercontent.com/11990514/178770472-850fa810-aaa2-41b9-9a81-989a96da9d95.mp4|https://user-images.githubusercontent.com/11990514/178770509-30da044d-4b56-4f25-aa8e-fb78d29d5aa8.mp4|
|By location|https://user-images.githubusercontent.com/11990514/178770654-7bef2d98-33a0-4eef-bae9-5682bede97f1.mp4|https://user-images.githubusercontent.com/11990514/178770719-80cb39ec-231b-4d51-bedb-00282632f600.mp4|






## Tests

1. Open a local room timeline (by clicking on the fab button from the DM room list, selecting one or several users, and pressing the "GO" button)
2. Send an event from the composer view (message, image or anything else)
3. The DM room should be created, the selected users should be invited and we can continue our discussion in that room.
4. Press the back button should redirect to the home screen, the local room is not displayed anymore.

## Tested devices

- OS version(s): API 32

## Checklist
